frontend/src/components/grant-form/index.tsx

```diff
@@ -76,18 +75,9 @@ export type GrantFormFields = {
   acceptedPrivacyPolicy: boolean;
 };
 
-export const MyGrantOrForm = () => {
+export const GrantSendForm = () => {
   const code = process.env.conferenceCode;
 
-  const { error, data } = useMyGrantQuery({
```

Comment:
When the user lands on `pycon.it/grants` page, if they have a grant, it automatically redirects to `profile/my-grant`. This component is only used when the user doesn't have a grant, so I removed the grant's fetching as it's no longer needed.